descartes_ikfast/test/descartes_ikfast_unit.cpp

```diff
@@ -0,0 +1,39 @@
+
+#include <gtest/gtest.h>
+#include <algorithm>
+#include <memory>
+#include <ostream>
+
+#include <descartes_ikfast_fanuc_m20ia10l_manipulator.h>
+
+TEST(DescartesIkFastUnit, Instantiation)
+{
+  Eigen::Isometry3d world_to_robot_base = Eigen::Isometry3d::Identity();
+  Eigen::Isometry3d tool0_to_tip = Eigen::Isometry3d::Identity();
+  descartes_ikfast_unit::FanucM20ia10lKinematicsD robot(world_to_robot_base, tool0_to_tip, nullptr, nullptr);
+
+  Eigen::Isometry3d pose;
+  std::vector<double> all_zeros = {0,0,0,0,0,0};
```

Comment:
Maybe replace use a different constructor here like:
` std::vector<double> all_zeros(6, 0.0);`

Reply:
I will make the change.

Reply:
Changed in latest push.